tests/countries/test_suriname.py

```diff
@@ -0,0 +1,236 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries import Suriname, SR, SUR
+from tests.common import CommonCountryTests
+
+
+class TestSuriname(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1972, 2050)
+        super().setUpClass(Suriname, years=years)
+
+    def test_country_aliases(self):
+        self.assertAliases(Suriname, SR, SUR)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Suriname(years=1971))
+
+    def test_new_years_day(self):
+        name = "Nieuwjaarsdag"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(1972, 2050)))
+
+    def test_day_of_liberation_and_renewal(self):
+        name = "Dag van Bevrijding en Vernieuwing"
+        self.assertHolidayName(
+            name, (f"{year}-02-25" for year in (*range(1981, 1993), *range(2012, 2021)))
+        )
+        self.assertNoHolidayName(name, range(1993, 2012), range(2021, 2050))
+
+    def test_holi(self):
+        name = "Holi-Phagwa"
+        self.assertHolidayName(name, range(2015, 2031))
+        self.assertHolidayName(
+            name,
+            "2019-03-21",
+            "2020-03-09",
+            "2021-03-28",
+            "2022-03-18",
+            "2023-03-07",
+            "2024-03-25",
+            "2025-03-14",
+        )
```

Comment:
```suggestion
        self.assertHolidayName(
            name,
            "2019-03-21",
            "2020-03-09",
            "2021-03-28",
            "2022-03-18",
            "2023-03-07",
            "2024-03-25",
            "2025-03-14",
        )
        self.assertHolidayName(name, range(2015, 2031))
```
Such order has already been widely established in other countries tests. (And a few similar places below)